### ISSUE TITLE
About FLOPs

### ISSUE THREAD

**Issue by AFeng-x** [OliverRensu/Shunted-Transformer#5]
Good work！And I wanna know how you calculate FLOPs of the model in your paper? Thanks!!!

**Comment by OliverRensu** [comment]
Hi, please refer mmcv flops count
https://github.com/open-mmlab/mmcv/blob/master/mmcv/cnn/utils/flops_counter.py

AFeng ***@***.***> 于2022年4月24日周日 17:09写道：

**Comment by Andy1621** [comment]
I remember that the mmcv FLOPs count does not support attention calculation, which will lead to lower FLOPs. I suggest tot use `fvcore` as in our [UniFormer](https://github.com/Sense-X/UniFormer/blob/ac62a49b40b79e614501905ad2f1c19bd4dcf12e/image_classification/main.py#L258-L259).

**Comment by OliverRensu** [comment]
Yes, we need to manually calculate the flops of Attention but it is not
very large with the input of 224x224.

Kunchang Li ***@***.***> 于2022年5月28日周六 02:35写道：
